q2_annotate/busco/busco.py

```diff
@@ -150,7 +283,33 @@ def _evaluate_busco(
         processing_func=_parse_busco_params, params=kwargs
     )
 
-    return _busco_helper(bins, common_args)
+    busco_results = _busco_helper(bins, common_args)
+    unbinned_percentage = {}
```

Comment:
This section needs to be conditional on `unbinned_contigs` - if the user did not input any this should be skipped.